src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -98,6 +98,28 @@ protected IReadOnlyList<WildcardMatcher> ParseSanitizeFieldNames(ConfigurationKe
 			}
 		}
 
+		protected IReadOnlyList<WildcardMatcher> ParseBaggageToAttach(ConfigurationKeyValue kv)
+			=> ParseWildcardMatcher(kv?.Value ?? DefaultValues.BaggageToAttach, "BaggageToAttach");
+
+		private IReadOnlyList<WildcardMatcher> ParseWildcardMatcher(string stringValue, string configName)
+		{
+			try
+			{
+				_logger?.Trace()?.Log("Try parsing {ConfigName}, values: {Values}", configName, stringValue);
+				var values = stringValue.Split(',').Where(n => !string.IsNullOrEmpty(n)).ToList();
+
+				var retVal = new List<WildcardMatcher>(values.Count);
```

Comment:
```suggestion
				var matchers = new List<WildcardMatcher>(values.Count);
```

This is nitpicky but `retVal` does not name what we are actually returning.